lib/ClangImporter/ImportDecl.cpp

```diff
@@ -8034,8 +8034,7 @@ void ClangImporter::Implementation::loadAllMembersOfObjcContainer(
   // Add the members now, before ~ImportingEntityRAII does work that might
   // involve them.
   for (auto member : members) {
-    IDC->addMember(member);
-  }
+    IDC->addMember(member);  }
```

Comment:
Typo here? } shouldn't be pulled up a line.

Reply:
Fixed